environment.yml

```
@@ -10,29 +10,37 @@ dependencies:
-      - torch==2.2.2
-      - openai-whisper==20250625
-      - pydub==0.25.1
-      - jiwer==3.0.3
-      - websockets==12.0
+      - certifi==2024.12.14
+      - click==8.1.8
+      - rich==13.9.4
+      - loguru==0.7.2
       - prometheus-client==0.20.0
-      - ruff==0.6.9
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)